packages/widgetbook/test/src/widgetbook_test.dart

```diff
@@ -40,11 +40,19 @@ void main() {
             'categories is empty',
             () {
               expect(
-                () => Widgetbook(
-                  addons: const [],
+                () => Widgetbook<ThemeData>(
+                  addons: [
+                    WidgetbookAddOn(
+                      name: 'A',
+                      builder: (context) => const SizedBox(),
+                    ),
+                  ],
+                  ],
                   categories: const [],
                   appInfo: appInfo,
-                  themes: themes,
+                  
+                  //addons:const []
+                //  themes: themes,
```

Comment:
This should also not be part of the PR.

Reply:
My bad, I had started working, on the other issues